net-print/gutenprint/gutenprint-5.3.4.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Ghostscript and cups printer drivers"
+HOMEPAGE="http://gutenprint.sourceforge.net"
+
+MY_P="${P/_/-}"
+S="${WORKDIR}/${MY_P}"
+SRC_URI="mirror://sourceforge/gimp-print/${MY_P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="cups gimp gtk nls readline ppds static-libs"
+RESTRICT="test"
```

Comment:
Why has this been done?

Reply:
That's an old thing: I haven't touched it because I don't know why it is here

Reply:
That's why when updating ebuilds such as this, always try to remove this restriction and test the ebuild. If it doesn't work, add a comment above the RESTRICT what exactly fails, and **preferably** either a link to a gentoo bug, or upstream issue about it. Test phase is pretty much the only reliable way for us to be sure the program works like upstream intends to.